src/apps/competitions/tests/test_phase_migration.py

```diff
@@ -0,0 +1,106 @@
+import datetime
+
+from django.test import TestCase
+from django.utils.timezone import now
+
+from competitions.tasks import do_phase_migrations
+from factories import UserFactory, CompetitionFactory, PhaseFactory, SubmissionFactory, SubmissionScoreFactory, \
+    CompetitionParticipantFactory
+
+twenty_minutes_ago = now() - datetime.timedelta(hours=0, minutes=20)
+five_minutes_ago = now() - datetime.timedelta(hours=0, minutes=5)
```

Comment:
Probably want these in the setup so they stay relative to the tests themselves, rather than fixed for all the tests?